### ISSUE TITLE
Training with custom dataset

### ISSUE THREAD

**Issue by Kir1s4me** [PRBonn/LiDiff#46]
Hello, I am currently training and testing this network with my custom dataset. But now the following issue has occurred

![Image](https://github.com/user-attachments/assets/d857a44d-ff99-47b4-a435-f94b1bdad6c8)
As shown in the figure , the inference results (T=1000, S=16) still exhibit a significant amount of noise, with the target point cloud obscured within the noise. I have tried modifying various parameters—such as increasing epochs, adjusting t_step, raising the learning rate, and reducing learning rate decay—but none have produced noticeable improvements. What do you think might be the issue? Which aspects should I focus on modifying? I’d greatly appreciate your insights. Thank you very much!!!

**Comment by nuneslu** [comment]
Hi! I refer you to [this](https://github.com/PRBonn/LiDiff/issues/14) issue. In the issue there is a long discussion regarding how to improve results for custom data. You can try the suggestions I mentioned there and in case it still does not work you can reopen this issue and I will be happy to help. :)
